sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/ApiCreateOrUpdateContent.Serialization.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ClientModel.Primitives;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ApiManagement;
+
+[CodeGenSerialization(nameof(TermsOfServiceUri), DeserializationValueHook = nameof(DeserializetermsOfServiceUrlValue))]
+public partial class ApiData : IUtf8JsonSerializable, IJsonModel<ApiData>
+{
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    internal static void DeserializetermsOfServiceUrlValue(JsonProperty property0, ref Uri termsOfServiceUrl)
+    {
+        if (property0.Value.ValueKind == JsonValueKind.Null)
+            return;
+        var str = property0.Value.GetString();
+        if (!string.IsNullOrEmpty(str))
+        {
+            if (Uri.TryCreate(str, UriKind.Absolute, out _))
+            {
+                termsOfServiceUrl = new Uri(str);
+            }
+        }
+    }
+}
+
+[CodeGenSerialization(nameof(ServiceUri), DeserializationValueHook = nameof(DeserializeServiceUriValue))]
+public partial class ApiData : IUtf8JsonSerializable, IJsonModel<ApiData>
```

Comment:
we can put them together, one class could have multiple attribute of this (the attribute is defined to allow multiple)